src/Test/L0/Plugin/TestFilePublisher/TestFilePublisherLogPluginL0.cs

```diff
@@ -332,7 +333,10 @@ public async Task TestFilePublisherLogPlugin_EnableForBuildPipeline()
                 { "agent.tempdirectory", new VariableValue("/tmp")},
```

Comment:
one more method encapsulating all this? Called CreateDefaultAgentContext? which internally adds all these props as well as calls AddPipelineDataIntoAgentContext().

Values not default will be overridden later